core/model-api/pom.xml

```diff
@@ -23,16 +23,13 @@
 				<artifactId>maven-jar-plugin</artifactId>
 				<executions>
 					<execution>
+						<phase>compile</phase>
```

Comment:
I thought this would fix an issue with building the test-jar on time, but it might have made it worse.